osu.Framework/Bindables/Bindable.cs

```diff
@@ -248,7 +248,8 @@ private void addWeakReference(WeakReference<Bindable<T>> weakReference)
         /// An object deriving T can be parsed, or a string can be parsed if T is an enum type.
         /// </summary>
         /// <param name="input">The input which is to be parsed.</param>
-        public virtual void Parse([CanBeNull] object input)
+        /// <param name="cultureInfo">The preferred culture formatting to be displayed.</param>
+        public virtual void Parse([CanBeNull] object input, [CanBeNull] CultureInfo cultureInfo = null)
```

Comment:
```suggestion
        /// <param name="provider">An object that provides culture-specific formatting information about <paramref name="input"/>.</param>
        public virtual void Parse([CanBeNull] object input, [CanBeNull] IFormatProvider provider = null)
```

https://learn.microsoft.com/en-us/dotnet/api/system.iparsable-1.parse?view=net-8.0#system-iparsable-1-parse(system-string-system-iformatprovider)